src/main/java/mate/academy/springbootwebgreqit/dto/CreateBookRequestDto.java

```diff
@@ -1,12 +1,16 @@
 package mate.academy.springbootwebgreqit.dto;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Positive;
 import lombok.Data;
 import java.math.BigDecimal;
 import java.util.List;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull(message = "Title is required")
+    @NotBlank(message = "Title cannot be empty")
```

Comment:
```suggestion
    @NotBlank(message = "Title cannot be empty")
```

add `@NotBlank` for required String fields

Add validation for price (`@NotNull and Min`)